projects.json

```
@@ -1133,6 +1147,13 @@
 				"OAuth"
 			]
 		},
+		{
+			"name": "theportalwiki.com",
+			"regex": "((?:www\\.)?theportalwiki\\.com)",
+			"articlePath": "/wiki/",
+			"scriptPath": "/w/",
+			"fullScriptPath": "https://theportalwiki.com/"
```

```suggestion
			"fullScriptPath": "https://theportalwiki.com/w/"
```
Additionally this wiki needs to be sorted under T, based on the name.